pkgs/development/python-modules/sumo/default.nix

```diff
@@ -31,30 +32,35 @@ buildPythonPackage rec {
     hash = "sha256-WoOW+JPo5x9V6LN+e8Vf3Q3ohHhQVK81s0Qk7oPn1Tk=";
   };
 
-  nativeBuildInputs = [ cython ];
+  nativeBuildInputs = [
+    cython
+  ];
 
```

Comment:
Are we not missing build-system setuptools here?

Reply:
Indeed, fixed. Rather cython is not needed.